### ISSUE TITLE
Change scribble/blueboxes to scribble/signature-box

### ISSUE THREAD

**Issue by jackfirth** [racket/scribble#206]
In racket/drracket#291 and racket/drracket#172, it was decided that "signature box" is a much better term than "blue box", and various parts of drracket's documentation were updated to use this phrase. Scribble's [`scribble/blueboxes`](https://docs.racket-lang.org/scribble/blueboxes.html) library uses the term "blue box" in its documentation and the names of its exported identifiers.

We should add a `scribble/signature-box` library that's the same as `scribble/blueboxes` but with changed names. Then we should deprecate the `scribble/blueboxes` library and point to the new library in its documentation.

**Comment by spdegabrielle** [comment]
@jackfirth I'd suggest this issue should be labelled 'good first issue' as it is a lot of steps but achievable for a beginner.

Steps to complete the task;
1. fork this repo`racket/scribble`
2. identify files that need to be changed <https://github.com/racket/scribble/search?q=blueboxes&unscoped_q=blueboxes>
3. create local copy on your PC
4. copy scribble-lib/scribble/blueboxes.rkt to scribble-lib/scribble/signature-box.rkt
5. copy `scribble-doc/scribblings/scribble/blueboxes.scrbl` to `scribble-doc/scribblings/scribble/signature-box.scrbl`
6. copy `scribble-lib/scribble/valid-blueboxes-info.rkt` to `scribble-lib/scribble/valid-signature-box-info.rkt`
7. change 'bluboxes' to 'signature-box' in files 4 ,5 , 6(above), `scribble-doc/scribblings/scribble/internals.scrbl`, `scribble-lib/scribble/contract-render.rkt` & `scribble-doc/scribblings/scribble/renderer.scrbl`.
8. delete `scribble-lib/scribble/blueboxes.rkt`, `scribble-doc/scribblings/scribble/blueboxes.scrbl` & `scribble-lib/scribble/valid-blueboxes-info.rkt`
9. test (is rebuilding racket/scribble sufficient?)
10. create PR for `racket/scribble`

**Comment by jackfirth** [comment]
@spdegabrielle Good idea! I don't seem to have issue-labeling powers though, and I don't know who would.

**Comment by jackfirth** [comment]
Thank you @rfindler!

**Comment by nloadholtes** [comment]
Hi, I gave this a PR a try but the tests weren't passing. This is the first time I've tried working with racket so I'm not sure what the error message means. I put the details in the PR #286
